API/Controllers/ProjectController.cs

```diff
@@ -792,6 +806,176 @@ public async Task<IActionResult> UnlikeProject(int projectId)
             return Ok(mapper.Map<ProjectLike, UserProjectLikeResourceResult>(projectLike));
         }
 
+        /// <summary>
+        ///     Categorize a project
+        /// </summary>
+        /// <param name="projectId"></param>
+        /// <param name="categoryId"></param>
+        /// <response code="200">Category was added to the project.</response>
+        /// <response code="401">User is not authorized to add the category to the project.</response>
+        /// <response code="404">Project or category was not found.</response>
+        /// <response code="409">Project is already categorized with the category.</response>
+        /// <returns></returns>
+        [HttpPost("category/{projectId}/{categoryId}")]
+        [Authorize]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
Try to specify the return type if there is one. This improves the swagger quality.

```suggestion
[ProducesResponseType(typeof(ProjectCategoryResourceResult), StatusCodes.Status200OK)]
```